### ISSUE TITLE
Switch brew formula to this repo.

### ISSUE THREAD

**Issue by ctopinka** [zsh-git-prompt/zsh-git-prompt#10]
Do you intend to do this? Would be nice to have this via Hombrew again. Thanks.

**Comment by thomasrebele** [comment]
I don't have any experience with homebrew. What exactly do you mean by "Switch brew formula to this repo"?

**Comment by rnapier** [comment]
Homebrew is a popular package manager on Mac. https://brew.sh

The current formula points to the olivierverdier repo: https://formulae.brew.sh/formula/zsh-git-prompt#default

It's not clear who maintains the Homebrew formula, however. (I expect this report should be made in the form of a PR to https://github.com/Homebrew/homebrew-core, rather than here.)

**Comment by thomasrebele** [comment]
Indeed, a PR to https://github.com/Homebrew/homebrew-core would be the right approach. It seems that this file has to be adapted: https://github.com/Homebrew/homebrew-core/blob/master/Formula/zsh-git-prompt.rb

**Comment by thomasrebele** [comment]
If no one is volunteering for adding this to homebrew again, I'll close the issue.

**Comment by christopinka** [comment]
Issue was here - https://github.com/Homebrew/homebrew-core/issues/73961
